www/http/validator.go

```diff
@@ -30,3 +34,41 @@ func (s *Server) GetValidatorHandler(w http.ResponseWriter, r *http.Request) {
 
 	s.writeHTML(w, tm.html())
 }
+
+// GetValidatorByNumberHandler returns a handler to get validator by number.
+func (s *Server) GetValidatorByNumberHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+
+	num, err := strconv.ParseInt(vars["number"], 10, 32)
+	if err != nil {
+		s.writeError(w, err)
+		return
+	}
+
+	if num > math.MaxInt32 || num < math.MinInt32 {
```

Comment:
This check is not necessary. HTTP is just a wrapper and should have any logic. 
Please don't forget to remove the tests for this block